site/content/docs/5.3/migration.md

```diff
@@ -11,11 +11,17 @@ toc: true
 
 ## v5.3.4
 
+<hr class="mb-4">
+
 ### Core
 
 - **Colors**
   - <span class="badge text-bg-warning">Warning</span> The dark mode red color hexadecimal value has been updated from `#f66` to `#ff4d4f` after a change in the design specifications to enhance the contrast for a better accessibility. This modification should be transparent for you except if you were using an hardcoded hexadecimal value directly in your websites.
 
+### CSS and Sass variables
+
+- <span class="badge text-bg-danger">Breaking</span> A parameter from the `focus-visible()` mixin has been removed. You don't need the `z-index` parameter anymore. Please reflect these modifications inside your website.
```

Comment:
Maybe explain this has an impact only if the project uses the mixin focus-visible() explicitly. 

Proposition:
<span class="badge text-bg-danger">Breaking</span> A parameter from the `focus-visible()` mixin has been removed. If you use the `focus-visible()` mixin in you scss files with a `z-index` as first parameter, please adapt your code.

Reply:
Yeah, I tried something based on your proposition in the last commit.